website/docs/r/logs_pipeline_order.html.markdown

```diff
@@ -35,14 +37,24 @@ No related field is available in  [Logs Pipeline API](https://docs.datadoghq.com
 
 ## Attributes Reference
 
-* `pipelines` - The `pipelines` list contains the IDs of resources created and imported by the [datadog_logs_pipeline](logs_pipeline.html#datadog_logs_pipeline).
+* `pipelines` - The `pipelines` list contains the IDs of resources created and imported by the 
+[datadog_logs_custom_pipeline](logs_custom_pipeline.html#datadog_logs_custom_pipeline) and 
+[datadog_logs_integration_pipeline](logs_integration_pipeline.html#datadog_logs_integration_pipeline).
 Updating the order of pipelines in this list reflects the application order of the pipelines. You cannot delete or create pipeline by deleting or adding IDs to this list.
 
 ## Import
 
 There must be at most one `datadog_logs_pipeline_order` resource. Pipeline order creation is not supported from logs config API. 
-You should import the `datadog_logs_pipeline_order` rather than create a pipeline order.
+You can import the `datadog_logs_pipeline_order` or create a pipeline order (which is actually doing the update operation).
 
 ```
 terraform import <datadog_logs_pipeline_order.name> <name>
 ```
+
+## Troubleshooting
+
+Sometimes when applying your `datadog_logs_pipeline_order` resource, you can get an error with status `422` 
+`Unprocessable Entity` with error message like `Cannot map pipelines to existing ones`, most likely the pipelines are 
+incompatible with the ones declared in your `datadog_logs_pipeline_order` resource. In this case, make
+sure that no pipeline gets added or deleted via other method (for example, through API call or from Datadog Logs
+configuration UI).
```

Comment:
Is there anything we can do to print a more helpful message to the user? Maybe even just printing the list of pipelines and asking users to confirm they have everything in that list defined in their config?

Reply:
Ahh ok. Can/do we provide a helpful error message in the code if this error is retrieved?

Reply:
I updated the method to do an extra call  a `GET` to get the order from the API, and print it along with the order from resource? what do you think? (it's a extra call to the API, but maybe it worths it since otherwise, user probably gonna need to do that anyways).

Reply:
I also remove this `troubleshooting` paragraph.